tracing/tracing.go

```diff
@@ -173,6 +178,10 @@ func GetTraceID(span ot.Span) string {
 		return spanContextType.TraceID().String()
 	case origlightstep.SpanContext:
 		return fmt.Sprintf("%x", spanContextType.TraceID)
+	case otelSpanContext:
+		// Use last 64 bits of the trace id (i.e. 16 hexadecimal characters)
+		traceId := spanContextType.TraceID().String()
+		return traceId[len(traceId)-16:]
```

Comment:
This can panic https://go.dev/play/p/r2HJ6le1rt8

Reply:
It can't - the trace id type is 16-byte array. But I changed it to simply return spanContextType.TraceID().String().